requirements.txt

```diff
@@ -1,4 +1,5 @@
-rtree==1.0.0
 jupyter-black==0.3.1
+jupytext==1.14.1
 matplotlib==3.5.3
-jupytext==1.14.1
\ No newline at end of file
+rasterstats==0.17.1
+rtree==1.0.0
```

Comment:
should we add:
ipykernel
pandas
geopandas
requests
aatoolbox (would this be changed to ocha-anticipy?)